docs/_optimist/guided_tour/step14/index.en.md

```diff
@@ -152,26 +147,24 @@ resources:
 ```
 
 - `ID of the resource`: Must be unique
-- `type`: Type of a resource, for example: OS::NEUTRON::SecurityGroup
+- `type`: Resource type, for example: OS::NEUTRON::SecurityGroup
     (for a security group) (required)
 - `properties`: A list of properties for resources (optional)
-- `metadata`: Metadata belonging to the resource  (optional)
-- `depends_on`: Resources that the resource depends on (optional)
-- `update_policy`: Specifies rules for updates, if required (optional)
-- `deletion_policy`: Specifies rules for the deletion. The options are
-    *Delete, Retain* and *Snapshot*. With heat\_template\_version 2016-10-14, you can also write them in lowercase.
-
-- `external_id`: You can use external IDs, if required
+- `metadata`: Metadata for the respective resource (optional)
+- `depends_on`: Various dependencies to other resources can be stored here (optional)
+- `update_policy`: Update rules can be defined here. As a prerequisite, ensure that the corresponding resource also supports this. (optional)
+- `deletion_policy`: Specifies rules for deletion. The options are
+    *Delete, Retain* and *Snapshot*. With heat\_template\_version 2016-10-14, you can also enter them in lowercase.
+- `external_id`: You can use external resource IDs, if required
 - `condition`: You can set specific conditions for this resource to be
-    created (optional)
+    created under (optional)
```

Comment:
is there a word missing after "under"?

Reply:
Nothing was missing here it referred to "conditions _under which_ the resource will be created" however, I did rewrite this for clarity.